packages/terra-action-footer/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Action Footer
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-action-footer.svg)](https://www.npmjs.org/package/terra-action-footer)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+The terra-action-footer component is a footer bar that contains sockets for placing actionable items such as buttons and hyperlinks. The default variant contains a start and end socket, while the centered variant has only a center socket. If no actions are provided, the footer bar collapses to a themeable height and maintains the top border.
```

Comment:
Can we use a word here other than `variant`? Makes me think there is a prop called `variant` :/

Reply:
Open to suggestions. `option`? `version`?

Reply:
[`Variation`?](http://engineering.cerner.com/terra-core/#/site/list)

![image](https://user-images.githubusercontent.com/17390173/37313789-bed3a6d4-261e-11e8-9c74-039eab4d452f.png)